about.html

```diff
@@ -0,0 +1,90 @@
+           <li class="desktop-link"><a href="#">News</a></li>
+           <li class="desktop-link"><a href="#" class="nav">CC Compaign</a></li>
+           </ul>
+               <div class="menu-icon">
+                   <div class="item-closed"><a href="#" class="closing"><i class="fa fa-times" aria-hidden="true"></i></a>
+                   </div>
+                   <div class="mobile-nav"><a href="./index.html" class="closing">Home</a></div>
+                   <div class="mobile-nav"><a href="./about.html" class="closing">About</a></div>
+                   <div class="mobile-nav"><a href="#" class="closing">Join</a></div>
+                 </div>
+           <div class="item-menu"><a href="#" class="openMenu"><i class="fa fa-bars"></i></a></div>
+           </nav>
+           </header>
+        <section class="about-me">   
+      <p class="small">"Hello! Sharing World"</p>
+        <h1 class="big">Creative Commons Global Summit 2015</h1>
+        <p class="lengthy">The Global Summit brings together the community of experts, academics, and activists who 
+            comprise the Creative Commons affiliate network in a different country every two years. This year, we 
+            hope to expand our invitation list including organizations and individuals.</p>
+            <p class="more-lengthy">The Global Summit brings together the community of experts, academics, and activists who 
+                comprise the Creative Commons affiliate network in a different country every two years. This year, we 
+                hope to expand our invitation list including organizations and individuals who want to work with us on shared projects 
+                that advance the cause of the Commons, free culture and open knowledge.So, if you're active and engaeged in the worlds of 
+                open content and knowledge - free software advocates,Wikipedians,Open Knowledge,galleries,libraries,museums,achieves,
+                governments and foundations.lawyers, and activists - we hope you'll consider joining us this year to build a stronger 
+                more vibrant commons together.</p>
+<p class="short">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+<a class="address"><u>ccsummit2015@cckorea.org</u></a>
+<h2 class="sec-heading">Creative Commons Global Summit 2015 Logo</h2>
+<hr class="underline">
+<div class="info"> The logo of CC Global Summit 2015 was decided through the logo competition from 8.June to 7. July.</div>
+<div class="more-info"> The logo of CC Global Summit 2015 was decided through the logo competition from 8.June to 7. July.
+                After online public voting and Selection Committee's judging process,the logo from Naresh Agrawal won
+                the prize. After consultation with Mr.Agrawal we optimized the prize-winning-logo for this year's summit as following:
+</div>
+<div class="small-logo"><img src="./images/CC_Global_Summit_2015 logo.png" alt="img"></div>
```

Comment:
- [ ] Good job so far :+1:  But this design is a bit different from the original design. :cry: Please use the same `height`, `width`, and `color` `opacity` for the elements as defined in the original design. You can compare the two screenshots below :point_down: to see the difference and fix the design :smile: 

#### Original design
![Screenshot from 2022-10-28 18-52-59](https://user-images.githubusercontent.com/6615394/198712591-466808db-6620-4463-bb6c-af0841d76188.jpg)